packages/playground/windows/playground/Playground.vcxproj

```diff
@@ -17,7 +17,6 @@
     <WindowsTargetPlatformVersion Condition=" '$(WindowsTargetPlatformVersion)' == '' ">10.0.18362.0</WindowsTargetPlatformVersion>
     <WindowsTargetPlatformMinVersion>10.0.16299.0</WindowsTargetPlatformMinVersion>
     <AppxPackageSigningEnabled>False</AppxPackageSigningEnabled>
```

Comment:
Does this get correctly overridden? A common MSBUILD pattern is to set a condition on a property, so that we only set the property if it is not yet set.

Reply:
Yes it should be good because it has 
```
<PropertyGroup Condition="'$(Configuration)'=='Release'">
    <AppxPackageSigningEnabled>True</AppxPackageSigningEnabled>
```
later in the file to override the global value for the prop when it is building in Release.